src/Gaufrette/Adapter/Dropbox.php

```diff
@@ -137,8 +137,8 @@ public function keys()
         foreach ($metadata['contents'] as $value) {
             $file = ltrim($value['path'], '/');
             $keys[] = $file;
-            if ('.' !== \Gaufrette\Util\Path::dirname($file)) {
-                $keys[] = \Gaufrette\Util\Path::dirname($file);
+            if ('.' !== $dirname = \Gaufrette\Util\Path::dirname($file)) {
```

Comment:
@akovalyov Still not convinced: from my understanding leading backslashes are unneeded and not recommended in the case of import (`use`) since imported namespaces are always fully qualified.
Do you mean I should import the function ?